.github/workflows/molecule.yml

```diff
@@ -0,0 +1,30 @@
+name: Test with Molecule
+on:
+  workflow_call:
+    inputs:
+      run_tags:
```

Comment:
Generally people use `run-tags` rather than `run_tags`

Reply:
I've renamed it now